src/less/navbar.less

```diff
@@ -561,6 +561,7 @@
 /////
 .navbar-inverse {
   background-color: @navbar-inverse-bg;
+  box-shadow: 0 1px 2px 0 rgba(0,0,0,.04), 0 2px 2px 0 rgba(0,0,0,.12);
```

Comment:
Do you want to apply this value to .navbar-default as well so both the light and dark navs have consistent shadows like Chi? Looks like .navbar-default's value is very similar so the change shouldn't have any noticeable difference.